src/shared/components/mutationTable/IMutationTableProps.ts

```diff
@@ -9,6 +9,7 @@ export interface IMutationTableProps extends IEnhancedReactTableProps<MutationTa
 {
     title?:string;
     samples?: ClinicalDataBySampleId;
+    onVisibleRowsChange?:(data:MutationTableRowData[]) => any;
```

Comment:
Any function will match () => void, so you should make the return value void if you are not using it.